src/test/Drips.t.sol

```diff
@@ -805,6 +828,161 @@ contract DripsTest is DSTest, PseudoRandomUtils {
         receiveDrips(receiver2, 55);
     }
 
+    function testDrippingFractions() public {
+        setDrips(
+            sender,
+            0,
+            2,
+            recv(receiver, 0, (1 * Drips.AMT_PER_SEC_MULTIPLIER) / cycleSecs + 1)
+        );
+        warpToCycleEnd();
+        receiveDrips(receiver, 1);
+        warpToCycleEnd();
+        receiveDrips(receiver, 1);
+        warpToCycleEnd();
+        receiveDrips(receiver, 0);
+    }
+
+    function testDrippingFractionsWithFundsEnoughForHalfCycle() public {
+        assertEq(cycleSecs, 10, "Unexpected cycle length");
+        uint256 onePerCycle = Drips.AMT_PER_SEC_MULTIPLIER / cycleSecs + cycleSecs;
```

Comment:
Because division always round down. `Drips.AMT_PER_SEC_MULTIPLIER / cycleSecs * cycleSecs` will be up to `cycleSecs - 1` lower than `Drips.AMT_PER_SEC_MULTIPLIER`.

It's an overkill though, `+ 1` is perfectly sufficient :sweat_smile: Fixed.